core/src/test/java/org/apache/calcite/test/enumerable/EnumerableBatchNestedLoopJoinTest.java

```diff
@@ -224,4 +228,104 @@
         .with(CalciteConnectionProperty.FORCE_DECORRELATE, forceDecorrelate)
         .withSchema("s", new ReflectiveSchema(schema));
   }
+
+
+  @Test void jdbcLeftBatchJoinTestSQL1() {
```

Comment:
Seems to me that if PR not only supports `EnumerableBatchNestedLoopJoin` in `JdbcToEnumerableConverter` but also tries to support dynamic parameters?

Reply:
yes，this pr is compatible with default dynamic parameters.